plugins/assimp/module/vtkF3DAssimpImporter.cxx

```diff
@@ -605,7 +605,7 @@ class vtkF3DAssimpImporter::vtkInternals
       }
 
       // convert materials to properties
-      this->Properties.resize(this->Scene->mNumMeshes);
+      this->Properties.resize(this->Scene->mNumMaterials);
```

Comment:
Fix the crash with `dwarf.x`

Reply:
do we need a test for this ?

Reply:
I don't think so. It was crashing when there was more materials than meshes. It's an edge case with weird datasets